### ISSUE TITLE
Cursor position in Edge browser for a touch screen device

### ISSUE THREAD

**Issue by pevgeniev** [hodgef/simple-keyboard#54]
**Simple-keyboard version**
2.15.2

**Describe the bug**
On a touch device in Edge browser, there is an issue with synchronizing the caret position. The problem is that Edge doesn't seem to allow placing the caret by clicking somewhere within the bounds of the text, but allows the user to drag the caret cursor. This 'dragging' doesn't fire touchend event, so the input's cursor and virtual keyboard caret go out of sync (the caretEventHandler is not executed). Does anyone know if this dragging behaviour in Edge (on touch device) can be prevented or changed to allow positioning within the text with a simple touch or if there is some Edge specific event for dragging the cursor and calliing caretEventHandler in it ?

**Comment by hodgef** [comment]
Hello @pevgeniev,

I would have to investigate this further. I should be providing an update later today. Thanks for submitting the [StackOverflow](https://stackoverflow.com/questions/54825567/edge-browser-caret-positioning-on-touch-device) issue by the way. I'll be keeping an eye on it.

Regards,
Francisco Hodge

**Comment by pevgeniev** [comment]
@hodgef thanks for looking it up. 
I've kind of worked around it for now by manually calling caretHandler from withing onKeyPress event handler (though caretHandler is private, so that might change in the API)

**Comment by hodgef** [comment]
Hello @pevgeniev,

Unfortunately, I was unable to properly emulate touch functionality in Edge. Unless I get a test device, it won't be something I'll be able to reproduce and fix.

I guess for Edge in touch devices I will be recommending [disableCaretPositioning](https://simple-keyboard.com/documentation/#disableCaretPositioning) for the time being.

I'll be closing this ticket for now, but if I come across a solution, I will update this ticket.

Regards,
Francisco Hodge

**Comment by hodgef** [comment]
Hello @pevgeniev,

I took a second look at this, as I was able to grab a friend's computer for testing. This seems to be a Edge bug. I tried just about any touch event I could think of, and still no event is fired when that selection pin is released on a touch device.

![edgebug](https://user-images.githubusercontent.com/25509135/53295155-3e024f80-37c3-11e9-8c31-c4b2a5b2c5b6.PNG)

Test sandbox: https://jsfiddle.net/zwj82ogb/

I see there has been an issue filed with the Edge team here:
https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/20615811/

I'll keep tracking that issue and keep the status as-is for now.

Regards,
Francisco Hodge
